File: 04 - listas/exercicios/python/lista.py
import os

from util import Util

lista1 = []
lista2 = []
lista_comuns = []
lista_disjuncao = []


objeto = Util()

objeto.popular_lista(lista1, 8)
objeto.popular_lista(lista2, 8)

os.system('cls')

objeto.exibir_lista("Lista 1", lista1)
objeto.exibir_lista("Lista 2", lista2)

os.system('pause')

objeto.localizar_comuns(lista_comuns, lista1, lista2)
objeto.exibir_lista("Numeros comuns nas listas", lista_comuns)

objeto.localizar_disjuncao(lista_disjuncao, lista1, lista2)
objeto.exibir_lista("Numeros disjuntos nas listas", lista_disjuncao)

File: 04 - listas/exercicios/c++/lista.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>

using namespace std;

#include "lista.h"
#include "util.h"

int main() {
    Lista *lista1 = NULL;
    Lista *lista2 = NULL;
    Lista *listaComuns = NULL;
    Lista *listaDisjuncao = NULL;

    lista1 = popularLista(lista1, 8);
    lista2 = popularLista(lista2, 10);

    system("cls");

    cout << "Lista 1" << endl;
    exibir(lista1);

    cout << "Lista 2" << endl;
    exibir(lista2);

    system("pause");

    listaComuns = localizarComuns(lista1, lista2);
    cout << "Números comuns nas listas" << endl;
    exibir(listaComuns);

    system("pause");

    listaDisjuncao = localizarDisjuncao(lista1, lista2);
    cout << "Números disjuntos nas listas" << endl;
    exibir(listaDisjuncao);

    



    return 1;
}